static/src/js/utils/getPermittedUser.js

```diff
@@ -0,0 +1,14 @@
+/**
+ *Logic to determine permittedUser based on environment
+ */
+const getPermittedUser = () => {
+  // For development environment
+  if (process.env.NODE_ENV === 'development') {
+    return 'typical'
+  }
+
+  // For SIT environment
+  return process.env.REACT_APP_SIT_PERMITTED_USER
```

Comment:
You should return `user.uid` here, so you'll want to pass in user to the function to destructure out the uid.

Reply:
updated this